src/components/ArticleVotes.jsx

```diff
@@ -0,0 +1,36 @@
+import { useState, useEffect} from "react"
+import { changeVote } from "../../api"
+
+function ArticleVotes({singleArticle}) {
+    const [optimisticVotes, setOptimisticVotes] = useState("")
```

Comment:
Consider making this have an initial state rather than waiting for the useEffect to set it on first render - this will reduce the number of useEffects you need (which we should try to use as less useEffects as possible to reduce the risk of side effects in our app)